forms-flow-web/src/components/Form/List.js

```diff
@@ -561,6 +580,30 @@ const List = React.memo((props) => {
   );
 });
 
+const ReleaseNoteModal = ({ showModal, onClose, title, content }) => {
+  return (
+      <Modal 
+        show={showModal}
+        onHide={onClose}
+        size="lg"
+        >
+        <Modal.Header style={{color: "#fff", backgroundColor: "#036"}}>
+          <Modal.Title>
+            <h3>{title}</h3>
+          </Modal.Title>
+        </Modal.Header>
+        <Modal.Body>
+            <div dangerouslySetInnerHTML={{ __html: content }} />
+        </Modal.Body>
+        <Modal.Footer>
+          <Button type="button" className="btn btn-default" onClick={onClose}>
+            <Translation>{(t) => t("Close")}</Translation>
+          </Button>
+        </Modal.Footer>
+      </Modal>
+  );
+};
+
```

Comment:
I would move this component to its own file. Maybe in `components/Form`?

Reply:
Moved it, Keep useEffect in the List.js as with the new suggestion to render form(using formio supported component) break when form unloads when hiding modal.